src/runtime/edbml/scripts/ts.ui.CalendarSpirit.edbml

```diff
@@ -51,11 +52,13 @@
 	}
 
 	function rendercell(cell) {
+		onrendercell(cell);
```

Comment:
I cleaned up this code because it looks unnecessarily complicated.

If you think that this is worse, I'm ok with finding a compromise.